reporting/src/main/resources/templates/summarized_email_report.mustache

```diff
@@ -155,13 +155,32 @@
             <br/>
             <span class="testResultSubTitle" align="center">Test Result Summary</span>
             <p/><p/>
-            <table>
+            <table class="reportTable">
                 <tr>
-                    <td valign="top" style="padding:0 10px 10px 5px">
-                        <img  src="{{summaryChart}}" alt="" border="0" align="left" height="300" width="300"/>
+                    <td valign="top" style="padding:10px 10px 10px 90px">
+                        <img  src="{{summaryChart}}" alt="" border="0" align="left" height="450" width="450"/>
                     </td>
-                    <td valign="middle" style="padding:0 10px 10px 100px">
-                      <img  src="{{historyChart}}" alt="" border="0" align="left" height="300" width="400"/>
+                    <td valign="middle" style="padding:10px 10px 10px 50px">
+                      <img  src="{{historyChart}}" alt="" border="0" align="left" height="450" width="550"/>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <pre>
+ <b>Test Failures:</b> Number of builds(build consist of one infra combination) failed
+                       due to test failures.
+ <b>Infrastructure Errors:</b> Number of builds failed due to infrastructure failures.
+                          (e.g: Product deployment failures etc.)
+ <b>Tests Passed: </b>Tests Passed infra combination.
+                        </pre>
```

Comment:
Number of successful builds (build consist of one infra combination) with no test failures.